Java-Majasdarbi/src/main/java/advanced/test/Exercise2/Apple.java

```diff
@@ -0,0 +1,14 @@
+package advanced.test.Exercise2;
+
+public class Apple {
+    public String id;
+
+    public Apple(String id) {
```

Comment:
Ābolam id? Uzdevumā gan bija pateikts `Create two empty classes Apple and Mushroom.`